csg.js

```diff
@@ -6481,6 +6531,11 @@ for solid CAD anyway.
                     }
                     thisside = sideTagToSideMap[nextsidetag];
                 } // inner loop
+                // due to the logic of CAG.fromPoints()
+                // move the first point to the last
+                if (connectedVertexPoints.length > 0) {
```

Comment:
ok, that is great then !
But perhaps for the future, could we also solve this by 'sorting' the points in a specific manner (using actual sort functions etc)